### ISSUE TITLE
Reference doc api links may not navigate to the correct documentation in the sidebar.

### ISSUE THREAD

**Issue by aYorky** [angular/angular#58053]
### Describe the problem that you experienced

When looking at the reference documentation, clicking on a property or function in the may not pull up the corresponding documentation in the sidebar. I've noticed this on several pages, though most recently on the FormGroup page. Likewise, clicking on one of the sidebar cards does not always highlight the correct item in the api. The offset seems to be fairly consistent per problematic page, though not all pages may have the same offset (i.e. clicking a property may pull up a doc for an item two down instead. This +2 offset will stay the same on that page, but another page may have a +1 offset). This may be related to properties that are not supposed to be publicly exposed (such as those starting with theta on the FormGroup page, which do not appear in the sidebar cards), but I can't say this definitively.

It seems that when a property is clicked and the card is brought up, the route registered in the browser correctly corresponds with the revealed card, but not with the actual property click. For example, if I click `readonly override valueChanges: Observable<TValue>;`, the route registered in the browser is instead `https://angular.dev/api/forms/FormGroup#setValidators`, and indeed, the setValidators card appears.

### Enter the URL of the topic with the problem

https://angular.dev/api/forms/FormGroup

### Describe what you want to experience that would fix the problem

Clicking on a property in the reference documentation should register the route for that property and expose the card for that property.

### Add a screenshot if that helps illustrate the problem

![Image](https://github.com/user-attachments/assets/b1a4692d-0268-4d5e-b5b8-7895b3bd0254)

**Comment by JeanMeche** [comment]
You're right, there seems to be some issue with the grouping (overloads are grouped}.

**Comment by aparzi** [comment]
this issue can be closed its PR has been merged
